package exception;


public class Demo
{
	public static void main(String[] args)
	{
		Test t=new Test();
		int result=Test1.addArray(new int[] {1,2,3,4});
		System.out.println(result);
		Exception e=new Exception();
		
	}

}
